File: project/game.py
import random

computer = random.randint(1, 3)

choices = {1: "Rock", 2: "Paper", 3: "Scissors"}

while True:
    user = int(input("Enter user choice: "))
    if user<1 or user>3:
        print("Invalid choice")
        continue
    print("User choose:", choices[user])

    print("Computer chose:", choices[computer])

    if computer == user:
        print("Tie")
   elif (computer == 1 and user == 2) or (computer == 2 and user == 3) or (computer == 3 and user == 1):
        print("User won and computer lost")
    else:
        print("User lost and computer won")
    play_again = input("Play again? (y/n): ").lower()
    if play_again != 'y':
        print("Thanks for playing!")

        break

File: project/game.java
import java.util.Random;
import java.util.Scanner;

public class RockPaperScissors {
    public static void main(String[] args) {

        Random random = new Random();
        Scanner sc = new Scanner(System.in);

        String[] choices = {"Rock", "Paper", "Scissors"};

        while (true) {
            int computer = random.nextInt(3) + 1; // 1 to 3

            System.out.print("Enter user choice (1-Rock, 2-Paper, 3-Scissors): ");
            int user = sc.nextInt();

            if (user < 1 || user > 3) {
                System.out.println("Invalid choice");
                continue;
            }

            System.out.println("User chose: " + choices[user - 1]);
            System.out.println("Computer chose: " + choices[computer - 1]);

            if (computer == user) {
                System.out.println("Tie");
            } else if (computer == 1 && user == 2 ||
                       computer == 2 && user == 3 ||
                       computer == 3 && user == 1) {
                System.out.println("User won and computer lost");
            } else {
                System.out.println("User lost and computer won");
            }

            System.out.print("Play again? (y/n): ");
            String playAgain = sc.next();

            if (!playAgain.equalsIgnoreCase("y")) {
                System.out.println("Thanks for playing!");
                break;
            }
        }
    }
}
